src/config/product.rhacm.js

```diff
@@ -102,7 +102,7 @@ const config = {
     {
       filters: [
         {
-          metric: RHSM_API_PATH_METRIC_TYPES.CORES,
+          metric: RHSM_API_PATH_METRIC_TYPES.VCPUS,
```

Comment:
required: looks like we missed the locale strings, need to add them for the new metric